doc/metals.txt

```diff
@@ -990,9 +990,33 @@ The following configuration values are available to be passed into
     node_command_mapping = "r",
     collapsed_sign = "▸",
     expanded_sign = "▾",
+    icons = {
+      enabled = false,
```

Comment:
It is disabled by default because I don't know how common is to have patched font installed.

Reply:
Yea, probably a good idea. After this is merged I can update the features list with a new screen shot showing this and with a message as a reminder that if someone is using a patched font they can enable this.